portfolio/src/main/webapp/gallery.html

```diff
@@ -7,16 +7,21 @@
   <link rel="stylesheet" href="style.css">
 </head>
 
-<body>
+  <body>
     <div id="content">
-        <h1 class="center"> Pictures of Jaden</h1>
-        <div class="img-format">
-          <img src="/Images/Jaden1.jpeg" alt="Image of Jaden" class="img">
-          <img src="/Images/Jaden2.jpeg" alt="Jaden Graduation"class="img">
-          <img src="/Images/Jaden3.jpeg" alt="Jaden in front of Bush Gardens christmas tree" class="img">
-          <img src="/Images/Jaden4.jpeg" alt="Jaden eating his first meal at hampton" class= "img">
-        </div>
+      <h1 class="center">Pictures of Jaden</h1>
+      <div class="img-format">
+        <img src="/Images/Jaden1.jpeg" title="Jaden's headshot" alt="Image of Jaden" class="img">
+        <img src="/Images/Jaden2.jpeg" title="Jaden at Graduation" alt="Jaden Graduation" class="img">
+        <img src="/Images/Jaden3.jpeg" title="Jaden at Bush Gardens" alt="Jaden in front of Bush Gardens christmas tree" class="img">
+        <img src="/Images/Jaden4.jpeg" title="Jaden eating at Hampton"alt="Jaden eating his first meal at hampton" class="img">
+        <img src="/Images/Jaden5.jpeg" title="Jaden and his mom in NYC" alt="Old picture of Jaden and his mom in NYC" class="img">
+        <img src="/Images/Jaden7.jpeg" title="Prague castle" alt="Picture of Prague castle" class="img">
+        <img src="/Images/Jaden6.jpeg" title="Jaden and Friends in Prague" alt="Picture of Jaden with Friends on Prague street" class="img">
+        <img src="/Images/Jaden8.jpeg" title="Turks and Caicos Beach" 
+          alt="Picture of Jaden on the beach in Turks and Caicos Islands." class="img">
```

Comment:
Good job breaking long lines. However, when indenting a line that's been broken, use a double indent (so four spaces instead of two in this case)

Reply:
Got it.